pre_build/cccp_index_pre_build_check.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
```

Comment:
Can we have some comments? `required_attrs` is obvious that it indicates required attributes to create jenkins job from container-index. But for functions can we document what parameters are expected and what is giong to be returned/done in the function? For example: in `def projectify` there are number of parameters and it looks like we expect `new_project` to be some kind of dictionary but, other parameters are probably strings.